### ISSUE TITLE
annotation `runtime/default` does not get applied to psp prometheus-operator-admission while updating

### ISSUE THREAD

**Issue by knrt10** [kinvolk/lokomotive#1128]
We have to delete the psp while updating to 0.5.0. Which is not a good UX. We should fix why this is happening

**Comment by invidian** [comment]
It seems PSP `prometheus-operator-admission` should be removed after applying the component, but it isn't for some reason. This prevents Helm from updating it later, which break the updates.

We should ensure that Helm hooks run properly and gets cleaned up as they should.

**Comment by invidian** [comment]
Also, if we update to Kubernetes 1.19.4, `runtime/default` PSP should work fine again.

**Comment by surajssd** [comment]
Is this valid, now?

**Comment by invidian** [comment]
Yes, we should investigate why PSP did not get removed as part of the update process.
